platform/ios/src/MGLFaux3DUserLocationAnnotationView.m

```diff
@@ -0,0 +1,522 @@
+#import "MGLFaux3DUserLocationAnnotationView.h"
+
+#import "MGLMapView.h"
+#import "MGLUserLocation_Private.h"
```

Comment:
Does this file use MGLUserLocation_Private.h for anything, or can it import MGLUserLocation instead? Ideally, a developer could theoretically implement anything that’s in this class in their own custom subclass of MGLUserLocationAnnotationView.